app/poros/station.rb

```diff
@@ -0,0 +1,13 @@
+class Station
+  attr_reader :name, :address, :fuel_type, :access_times, :city, :state, :zip
+
+  def initialize(data)
+    @name = data[:station_name]
+    @address = data[:street_address]
```

Comment:
As mentioned in the comment in the controller, here you could concatenate the address components if you wanted to. It's an operation I would expect to see somewhere in the PORO rather than in the controller, as it's manipulating attributes of this domain object.